data/json/itemgroups/tools.json

```diff
@@ -154,6 +154,16 @@
     ]
   },
   {
+    "id" : "tools_lighting_industrial",
```

Comment:
This is probably a good example of unavoidable duplication. I can see why you don't want to inherit from `tools_lighting` which contains candles and such